### PR TITLE
feat: Support DENO_INSTALL in the powershell installer

### LINKED ISSUE #76: Support for DENO_INSTALL in the PowerShell script

### PR DESCRIPTION
**Pull request by nayeemrmn**
Closes #76.

### CODE REVIEW

File: install_test.ps1 [hunk 1]
```diff
@@ -17,29 +17,27 @@ if (!(Get-Module PSScriptAnalyzer -ListAvailable)) {
...
+  ~\deno-0.13.0\bin\deno.exe --version
 } else {
-  ~/.deno/bin/deno --version
+  ~/deno-0.13.0/bin/deno --version
 }
 if (!($DenoVersion -like '*0.13.0*')) {
   throw $DenoVersion
 }
-
-# Test we can install the latest version.
```
Comment (ry) [inline install_test.ps1]: why remove this?

Comment (nayeemrmn) [reply install_test.ps1]: It's just moved up to match the shell version. Sorry for the confusion.

Comment (nayeemrmn) [reply install_test.ps1]: I think you meant to click on the `Remove-Item`. I don't know why I removed those! Restored.

**Comment by nayeemrmn** [comment]
@MarkTiedemann @piscisaureus Please review.

**Comment by MarkTiedemann** [comment]
@nayeemrmn LGTM

**Pull request merged**